deploy/charts/istio-csr/README.md

```diff
@@ -459,6 +459,25 @@ tolerations:
   value: master
   effect: NoSchedule
 ```
+#### **topologySpreadConstraints** ~ `array`
+> Default value:
+> ```yaml
+> []
+> ```
+
+List of Kubernetes TopologySpreadConstraints. For more information, see [TopologySpreadConstraint v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#topologyspreadconstraint-v1-core).  
+For example:
+
+```yaml
+topologySpreadConstraints:
+- maxSkew: 2
+  topologyKey: topology.kubernetes.io/zone
+  whenUnsatisfiable: ScheduleAnyway
+  labelSelector:
+    matchLabels:
+      app.kubernetes.io/instance: cert-manager
+      app.kubernetes.io/component: controller
+```
```

Comment:
This is the same example as [we added to trust-manager](https://docs.venafi.cloud/vaas/k8s-components/c-tm-helmvalues/#topologyspreadconstraints), but it looks like it's copy-pasted from cert-manager.
The matchLabels  should have trust-manager app labels, I think.

Reply:
You're absolutely right! I've fixed the example, will raise a quick PR for trust-manager too!